src/TransactionTestsFiller/ttGasLimit/TransactionWithGasLimitOverflowZeros64Filler.json

```diff
@@ -0,0 +1,21 @@
+{
+    "TransactionWithGasLimitOverflowZeros64" :
+	{
+		"expectException" : 
+        {
+            ">=Frontier" : "InvalidGasLimit4"
```

Comment:
the exception here is the same as in overflow, but shouldn't it actually throw leading zeroes exception?

Reply:
For the input "0x000000000000000001", we were expecting the leading-zero-bytes exception:
> rlp: non-canonical integer (leading zero bytes) for uint64, decoding into (types.Transaction)(types.LegacyTx).Gas

And we are getting the input-too-long exception:
> rlp: input string too long for uint64, decoding into (types.Transaction)(types.LegacyTx).Gas

But after thinking about it, I don't think this is really an issue, both seem to be valid exceptions for the test case.